plugins/splunk-on-call/config.d.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  /**
+   * Splunk On Call Plugin specific configs
+   */
+  splunkOnCall?: {
+    /**
+     * @visibility frontend
+     */
+    eventsRestEndpoint: string;
```

Comment:
should it be optional? does it have a sensible fallback?

Reply:
Yep, makes sense to have this optional too